xapian-letor/tests/api_letor.cc

```diff
@@ -25,7 +25,7 @@
 #include <cstdlib>
 #include <fstream>
 #include <sstream>
-
+#include <iostream>
```

Comment:
It looks to me like this isn't actually used, though testcases really shouldn't be using `cout`, etc directly anyway.

Reply:
yeah will fix it.